services/wireguard/service/service_unix.go

```diff
@@ -100,11 +115,14 @@ func (manager *Manager) ProvideConfig(sessionConfig json.RawMessage, traversalPa
 	}
 
 	destroy := func() {
+		if err := dnsServer.Stop(); err != nil {
+			_ = log.Error("failed to stop DNS server", err)
+		}
 		if err := manager.natService.Del(natRule); err != nil {
-			log.Error("failed to delete NAT forwarding rule: ", err)
+			_ = log.Error("failed to delete NAT forwarding rule: ", err)
```

Comment:
Is this change really needed here? If not using variable, lets not define it as `_` at all

Reply:
`log.Error` return and error and linter complain about it. Explicitly marking that we don't care about this error.

Reply:
I would say, lets change our abstraction `log.Logger` instead

Reply:
Changed.